libs/sdk-ui-charts/src/highcharts/chartTypes/_chartCreators/getChartOrientationConfiguration.ts

```diff
@@ -0,0 +1,37 @@
+// (C) 2023 GoodData Corporation
+
+import { IChartConfig } from "../../../interfaces/index.js";
+import { HighchartsOptions } from "../../lib/index.js";
+import { IChartOptions } from "../../typings/unsafe.js";
+import { isWaterfall } from "../_util/common.js";
+
+export function getChartOrientationConfiguration(
```

Comment:
This function should be typed somehow. Isn't there some existing type that can be used?

Reply:
Isn't this the `HighchartsOptions`?

Reply:
yes, this function returns the HighchartsOptions type. I have added the type for it. Thanks!